tsfresh/transformers/feature_selector.py

```diff
@@ -82,7 +84,7 @@ def fit(self, X, y):
             y = pd.Series(y.copy())
 
         df_bh = check_fs_sig_bh(X, y, self.settings)
-        self.relevant_features = df_bh.loc[df_bh.rejected].Feature
+        self.relevant_features = df_bh.loc[df_bh.rejected].p_value
```

Comment:
But the change of the API does make sense, because `self.relevant_features` contains the relevant features both in its index and its values.

We could document this change for version 0.5 thoroughly and maybe even think about combining it with a diagram explaining the Benjamini-Yekutieli (cf. Fig 2 in ACML paper).

>